run

```diff
@@ -0,0 +1 @@
+node src/build/run.mjs $@
```

Comment:
why not have that be a script in package.json?

Reply:
Tbh, because I didn't immediately find a way to pass all command-line arguments on to a package json script. It seemed to only use the first one